docs/content/configuration/identity-providers/openid-connect/provider.md

```diff
@@ -553,6 +551,7 @@ Automatically adds the origin portion of all redirect URI's on all clients to th
 localhost.
 
 ### clients
+{{< confkey type="list(object)" required="yes" >}}
```

Comment:
```suggestion

{{< confkey type="list(object)" required="yes" >}}
```

Reply:
I swear there was only the tag change. Apologies.

Reply:
Nah all good, that's what review is for.